mscore/data/shortcuts.xml

```diff
@@ -491,6 +491,14 @@
     <seq>+</seq>
     </SC>
   <SC>
+    <key>pad-raise</key>
+    <seq>NumPad++</seq>
```

Comment:
and I just tried: it does conflict with the numpad + to create ties, so this is a no-go, to me at least